### DISCUSSION TITLE
show watched history

### DISCUSSION THREAD

**Discussion by EdeAbreu23** [trakt/showly#439] [Ideas / Feature Requests]
i would like to be able to see my last watched tv show/movies within the app
![image](https://user-images.githubusercontent.com/7027367/109006479-0759bb00-76a3-11eb-8aed-2439c0b28f56.png)

**Comment by Matth7878** [comment]
For what it's worth I opened issue #563 which imho would partially address your point. Hope it would be implemented as it would make collection tab a lot more useful.
Didn't see this discussion before doing it.
